25-minimize.ks

```diff
@@ -65,6 +65,7 @@ echo " * locking root account"
 passwd -l root
 
 echo " * cleaning up yum cache and removing rpm database"
+rpm -qa > /usr/packages_list
```

Comment:
I realized we don't carry list of installed packages anymore (we delete RPM database to save some space). This gives a list that is reported by foreman-debug tool.